### PR TITLE
Move extensions projects to sdk/extensions directory

### PR DESCRIPTION
**Pull request by pakrym**
As discussed offline moving all extensions to the same location.

### CODE REVIEW

**Review by tg-msft — APPROVED** [review]

File: sdk/extensions/ci.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - master
+    - feature/*
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/extensions/
+
+stages:
+- template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
```
Comment (weshaggard) [inline sdk/extensions/ci.yml]: What about other extensions like Microsoft.Extensions.Azure? Should they also be moved into here?

Comment (pakrym) [reply sdk/extensions/ci.yml]: Yes, I'm planning to do that too. Maybe even rename it.

File: sdk/extensions/tests.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+trigger: none
```
Comment (weshaggard) [inline sdk/extensions/tests.yml]: We have pipeline generation tooling that should automatically pick-up and setup a pipeline for the ci.yml file but we currently still have to manually setup the tests pipeline. Work with @danieljurek if you need help setting that up.

Comment (pakrym) [reply sdk/extensions/tests.yml]: Done.

**Pull request merged**